tests/test_numpy-vectorize.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+try:
+    import numpy as np
+except ImportError:
+    np = None
+
+requires_numpy = pytest.mark.skipif(not np, reason="numpy is not installed")
```

Comment:
Seeing as this is repeated, could it be moved to `conftest.py` (can export via `pytest_namespace()` there and then in test files use it as `@pytest.requires_numpy`)